README.md

```diff
@@ -29,12 +29,12 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 - Windows 11 or Windows 10 v1903 (18362) or newer.
 - Our installer will install the following items:
    - [.NET Core 3.1.22 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-3.1.22-windows-x64-installer) or a newer 3.1.x runtime. This is needed currently for the Settings application.
```

Comment:
```suggestion
    - [.NET Core 3.1.22 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-3.1.22-windows-x64-installer) or a newer 3.1.x runtime. (This is needed currently for the Settings application.)
```

Reply:
I didn't understand this suggestion, especially since I think all the points need to be on the same level for markdown.